CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ This file documents any relevant changes done to ViUR server since version 2.
 
 ## [develop] - Current development version
 
+### Added
+- translate bone params _category_ and _tooltip_ in json renderer
+
 
 ## [2.5.0] Vesuv - 2019-06-07
 
 ### Added
 
-- updateLevel for relationalBones, to exclude specific bones from the automatic background update ([#170](https://github.com/viur-framework/server/pull/170))
+- updateLevel for relationalBones, to exclude specific bones from thpwd
```

Comment:
thpwd...e? Seems you lost focus from a terminal window ;-)

Reply:
it seems so... :/